app/main.py

```diff
@@ -1,15 +1,12 @@
 def copy_file(command: str) -> None:
-    parts = command.split()
-
-    if len(parts) == 3 and parts[0] == "cp":
-        source_file, copied_file = parts[1], parts[2]
-
-        if source_file != copied_file:
-            with (
-                open(source_file, "r") as read_file,
-                open(copied_file, "w") as write_file,
-            ):
-                write_file.write(read_file.read())
+    _, source_file, copied_file = command.split()
```

Comment:
what if `_ != "cp"`?)

Reply:
> what if `_ != "cp"`?)

But we don`t use "cp" here. Why we need to save it in variable?

Reply:
exactly what I said. Why we don't use it?)